recipes/mne-microstates/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mne-microstates" %}
+{% set version = "0.3" %}
+{% set pypi_filename = "mne_microstates" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ pypi_filename }}-{{ version }}.tar.gz
+  sha256: cafde4b3f8384a32ed8bc8721c0b51dcbb46326b54e9b631d6d379b02f4fdf9f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Reply:
This was taken straight from the template btw 🤔

Reply:
Ahh shoot yes I forgot to change that.